Address Safer CPP failures in WebKit/Shared/

cdumez opened the pull request:
#### 72acb3c653a2944adb8db03d0e1413bfe328ea62
<pre>
Address Safer CPP failures in WebKit/Shared/
<a href="https://bugs.webkit.org/show_bug.cgi?id=289660">https://bugs.webkit.org/show_bug.cgi?id=289660</a>

Reviewed by Darin Adler.

* Source/WebKit/SaferCPPExpectations/MemoryUnsafeCastCheckerExpectations:
* Source/WebKit/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebKit/Shared/API/c/WKArray.cpp:
(WKArrayGetItemAtIndex):
* Source/WebKit/Shared/API/c/WKContextMenuItem.cpp:
(WKContextMenuItemCreateAsAction):
(WKContextMenuItemCreateAsCheckableAction):
(WKContextMenuItemCreateAsSubmenu):
(WKContextMenuItemSeparatorItem):
(WKContextMenuCopySubmenuItems):
(WKContextMenuItemGetUserData):
(WKContextMenuItemSetUserData):
* Source/WebKit/Shared/API/c/WKDictionary.cpp:
(WKDictionaryCreate):
(WKDictionaryGetItemForKey):
(WKDictionaryCopyKeys):
* Source/WebKit/Shared/API/c/WKErrorRef.cpp:
(WKErrorCopyDomain):
(WKErrorGetErrorCode):
(WKErrorCopyFailingURL):
(WKErrorCopyLocalizedDescription):
* Source/WebKit/Shared/API/c/WKImage.cpp:
(WKImageGetSize):
* Source/WebKit/Shared/API/c/WKMutableDictionary.cpp:
(WKDictionarySetItem):
* Source/WebKit/Shared/API/c/WKSecurityOriginRef.cpp:
(WKSecurityOriginCreateFromString):
(WKSecurityOriginCreateFromDatabaseIdentifier):
(WKSecurityOriginCreate):
* Source/WebKit/Shared/API/c/WKSharedAPICast.h:
(WebKit::toAPILeakingRef):
(WebKit::toImpl):
(WebKit::toProtectedImpl):
(WebKit::toWTFString):
* Source/WebKit/Shared/API/c/WKString.cpp:
(WKStringIsEmpty):
(WKStringGetLength):
(WKStringGetCharacters):
(WKStringGetMaximumUTF8CStringSize):
(WKStringGetUTF8CStringImpl):
(WKStringIsEqual):
(WKStringIsEqualToUTF8CString):
(WKStringIsEqualToUTF8CStringIgnoringCase):
(WKStringCopyJSString):
* Source/WebKit/Shared/API/c/WKType.cpp:
(WKGetTypeID):
* Source/WebKit/Shared/API/c/WKURL.cpp:
(WKURLCreateWithBaseURL):
(WKURLIsEqual):
(WKURLCopyHostName):
(WKURLCopyScheme):
(WKURLCopyPath):
(WKURLCopyLastPathComponent):
* Source/WebKit/Shared/API/c/WKUserContentURLPattern.cpp:
(WKUserContentURLPatternCreate):
(WKUserContentURLPatternMatchesURL):
* Source/WebKit/Shared/API/c/cf/WKStringCF.mm:
(WKStringCreateWithCFString):
(WKStringCopyCFString):
* Source/WebKit/Shared/API/c/cf/WKURLCF.mm:
(WKURLCreateWithCFURL):
* Source/WebKit/Shared/API/c/mac/WKWebArchiveRef.cpp:
(WKWebArchiveCreate):
(WKWebArchiveCreateWithData):
(WKWebArchiveCreateFromRange):
(WKWebArchiveCopyMainResource):
(WKWebArchiveCopySubresources):
(WKWebArchiveCopySubframeArchives):
(WKWebArchiveCopyData):
* Source/WebKit/Shared/API/c/mac/WKWebArchiveResource.cpp:
(WKWebArchiveResourceCreate):
(WKWebArchiveResourceCopyData):
(WKWebArchiveResourceCopyURL):
(WKWebArchiveResourceCopyMIMEType):
(WKWebArchiveResourceCopyTextEncoding):
* Source/WebKit/Shared/APIWebArchiveResource.h:
* Source/WebKit/Shared/BlobDataFileReferenceWithSandboxExtension.h:
* Source/WebKit/Shared/Cocoa/WKNSData.mm:
(-[WKNSData dealloc]):
* Source/WebKit/Shared/Cocoa/WKNSString.mm:
* Source/WebKit/Shared/Cocoa/WKNSURL.mm:
(-[WKNSURL _web_createTarget]):
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithInProcessRenderingBackingStore.mm:
(WebKit::RemoteLayerWithInProcessRenderingBackingStore::frontBufferMayBeVolatile const):
(WebKit::RemoteLayerWithInProcessRenderingBackingStore::createContextAndPaintContents):
(WebKit::RemoteLayerWithInProcessRenderingBackingStore::createFlusher):
(WebKit::RemoteLayerWithInProcessRenderingBackingStore::setBufferNonVolatile):
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithRemoteRenderingBackingStore.mm:
(WebKit::RemoteLayerWithRemoteRenderingBackingStore::RemoteLayerWithRemoteRenderingBackingStore):
* Source/WebKit/Shared/WebBackForwardListItem.h:
(isType):
* Source/WebKit/Shared/WebGeolocationPosition.h:
(isType):
* Source/WebKit/UIProcess/API/APIApplicationManifest.h:
* Source/WebKit/UIProcess/API/APIAttachment.h:
* Source/WebKit/UIProcess/API/APIContentRuleList.h:
* Source/WebKit/UIProcess/API/APIContentRuleListAction.h:
* Source/WebKit/UIProcess/API/APIContentRuleListStore.h:
* Source/WebKit/UIProcess/API/APIContentWorld.h:
* Source/WebKit/UIProcess/API/APIContextMenuElementInfo.h:
* Source/WebKit/UIProcess/API/APIContextMenuElementInfoMac.h:
* Source/WebKit/UIProcess/API/APIDataTask.h:
* Source/WebKit/UIProcess/API/APIFeature.h:
* Source/WebKit/UIProcess/API/APIFrameInfo.h:
* Source/WebKit/UIProcess/API/APIFrameTreeNode.h:
* Source/WebKit/UIProcess/API/APIHTTPCookieStore.h:
* Source/WebKit/UIProcess/API/APIHitTestResult.h:
* Source/WebKit/UIProcess/API/APIInspectorExtension.h:
* Source/WebKit/UIProcess/API/APIMessageListener.h:
* Source/WebKit/UIProcess/API/APINavigation.h:
* Source/WebKit/UIProcess/API/APINavigationAction.h:
* Source/WebKit/UIProcess/API/APINavigationData.h:
* Source/WebKit/UIProcess/API/APINavigationResponse.h:
* Source/WebKit/UIProcess/API/APIOpenPanelParameters.h:
* Source/WebKit/UIProcess/API/APIPageConfiguration.h:
* Source/WebKit/UIProcess/API/APIProcessPoolConfiguration.h:
* Source/WebKit/UIProcess/API/APIResourceLoadInfo.h:
* Source/WebKit/UIProcess/API/APIResourceLoadStatisticsFirstParty.h:
* Source/WebKit/UIProcess/API/APIResourceLoadStatisticsThirdParty.h:
* Source/WebKit/UIProcess/API/APISessionState.h:
* Source/WebKit/UIProcess/API/APITargetedElementInfo.h:
* Source/WebKit/UIProcess/API/APITargetedElementRequest.h:
* Source/WebKit/UIProcess/API/APITextRun.h:
* Source/WebKit/UIProcess/API/APIUserInitiatedAction.h:
* Source/WebKit/UIProcess/API/APIWebAuthenticationPanel.h:
* Source/WebKit/UIProcess/API/APIWebPushDaemonConnection.h:
* Source/WebKit/UIProcess/API/APIWebPushMessage.h:
* Source/WebKit/UIProcess/API/APIWebPushSubscriptionData.h:
* Source/WebKit/UIProcess/API/APIWebsiteDataRecord.h:
* Source/WebKit/UIProcess/API/APIWebsitePolicies.h:
* Source/WebKit/UIProcess/API/APIWindowFeatures.h:
* Source/WebKit/UIProcess/Authentication/AuthenticationChallengeProxy.h:
(isType):
* Source/WebKit/UIProcess/Authentication/AuthenticationDecisionListener.h:
(isType):
* Source/WebKit/UIProcess/Authentication/WebCredential.h:
(isType):
* Source/WebKit/UIProcess/Authentication/WebProtectionSpace.h:
(isType):
* Source/WebKit/UIProcess/Downloads/DownloadProxy.h:
(isType):
* Source/WebKit/UIProcess/GeolocationPermissionRequestProxy.h:
(isType):
* Source/WebKit/UIProcess/Inspector/WebInspectorUIProxy.h:
(isType):
* Source/WebKit/UIProcess/MediaKeySystemPermissionRequest.h:
(isType):
* Source/WebKit/UIProcess/Notifications/WebNotification.h:
(isType):
* Source/WebKit/UIProcess/QueryPermissionResultCallback.h:
(isType):
* Source/WebKit/UIProcess/SpeechRecognitionPermissionRequest.h:
(isType):
* Source/WebKit/UIProcess/UserContent/WebUserContentControllerProxy.h:
(isType):
* Source/WebKit/UIProcess/UserMediaPermissionCheckProxy.h:
(isType):
* Source/WebKit/UIProcess/UserMediaPermissionRequestProxy.h:
(isType):
* Source/WebKit/UIProcess/WebBackForwardList.h:
(isType):
* Source/WebKit/UIProcess/WebContextMenuListenerProxy.h:
(isType):
* Source/WebKit/UIProcess/WebFormSubmissionListenerProxy.h:
(isType):
* Source/WebKit/UIProcess/WebFramePolicyListenerProxy.h:
(isType):
* Source/WebKit/UIProcess/WebGeolocationManagerProxy.h:
(isType):
* Source/WebKit/UIProcess/WebPreferences.h:
(isType):
* Source/WebKit/UIProcess/WebProcessPool.h:
(isType):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h:
(isType):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.h:
(isType):
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeContext.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeContext.mm:
(WebKit::RemoteLayerTreeContext::ensureProtectedRemoteRenderingBackendProxy):

Canonical link: <a href="https://commits.webkit.org/292130@main">https://commits.webkit.org/292130@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4b3f85d7f7774a0a99aa54ae99e8b4ec15ab1247

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/95073 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/14668 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/4525 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/100103 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/45572 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/14957 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/23090 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/72515 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/45572 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/98074 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/132/builds/11164 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/85836 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/52843 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/10874 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | [  ~~🛠 wpe-cairo~~](https://ews-build.webkit.org/#/builders/65/builds/44911 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/81067 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/136/builds/3658 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/102149 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/22115 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/16136 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/81520 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/22362 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/81858 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/80911 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/20215 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/25469 "Passed tests") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/2872 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/15357 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/22088 "Built successfully") | [  ~~🛠 mac-safer-cpp~~](https://ews-build.webkit.org/#/builders/120/builds/27218 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/21745 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/25220 "Built successfully") | | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/124/builds/23486 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

darinadler commented on Source/WebKit/Shared/Cocoa/WKNSString.mm:
```diff
@@ -35,7 +35,7 @@ @implementation WKNSString
 {
-    const String& string = downcast<API::String>(&self._apiObject)->string();
+    const String& string = RefPtr { downcast<API::String>(&self._apiObject) }->string();
     return (NSString *)string.createCFString().leakRef();
```
This looks a bit iffy to me. Does the string function return an internal reference? I think the local should be of type `String` or we should just write this without a local variable.

cdumez committed:
Address Safer CPP failures in WebKit/Shared/
https://bugs.webkit.org/show_bug.cgi?id=289660

Reviewed by Darin Adler.

* Source/WebKit/SaferCPPExpectations/MemoryUnsafeCastCheckerExpectations:
* Source/WebKit/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebKit/Shared/API/c/WKArray.cpp:
(WKArrayGetItemAtIndex):
* Source/WebKit/Shared/API/c/WKContextMenuItem.cpp:
(WKContextMenuItemCreateAsAction):
(WKContextMenuItemCreateAsCheckableAction):
(WKContextMenuItemCreateAsSubmenu):
(WKContextMenuItemSeparatorItem):
(WKContextMenuCopySubmenuItems):
(WKContextMenuItemGetUserData):
(WKContextMenuItemSetUserData):
* Source/WebKit/Shared/API/c/WKDictionary.cpp:
(WKDictionaryCreate):
(WKDictionaryGetItemForKey):
(WKDictionaryCopyKeys):
* Source/WebKit/Shared/API/c/WKErrorRef.cpp:
(WKErrorCopyDomain):
(WKErrorGetErrorCode):
(WKErrorCopyFailingURL):
(WKErrorCopyLocalizedDescription):
* Source/WebKit/Shared/API/c/WKImage.cpp:
(WKImageGetSize):
* Source/WebKit/Shared/API/c/WKMutableDictionary.cpp:
(WKDictionarySetItem):
* Source/WebKit/Shared/API/c/WKSecurityOriginRef.cpp:
(WKSecurityOriginCreateFromString):
(WKSecurityOriginCreateFromDatabaseIdentifier):
(WKSecurityOriginCreate):
* Source/WebKit/Shared/API/c/WKSharedAPICast.h:
(WebKit::toAPILeakingRef):
(WebKit::toImpl):
(WebKit::toProtectedImpl):
(WebKit::toWTFString):
* Source/WebKit/Shared/API/c/WKString.cpp:
(WKStringIsEmpty):
(WKStringGetLength):
(WKStringGetCharacters):
(WKStringGetMaximumUTF8CStringSize):
(WKStringGetUTF8CStringImpl):
(WKStringIsEqual):
(WKStringIsEqualToUTF8CString):
(WKStringIsEqualToUTF8CStringIgnoringCase):
(WKStringCopyJSString):
* Source/WebKit/Shared/API/c/WKType.cpp:
(WKGetTypeID):
* Source/WebKit/Shared/API/c/WKURL.cpp:
(WKURLCreateWithBaseURL):
(WKURLIsEqual):
(WKURLCopyHostName):
(WKURLCopyScheme):
(WKURLCopyPath):
(WKURLCopyLastPathComponent):
* Source/WebKit/Shared/API/c/WKUserContentURLPattern.cpp:
(WKUserContentURLPatternCreate):
(WKUserContentURLPatternMatchesURL):
* Source/WebKit/Shared/API/c/cf/WKStringCF.mm:
(WKStringCreateWithCFString):
(WKStringCopyCFString):
* Source/WebKit/Shared/API/c/cf/WKURLCF.mm:
(WKURLCreateWithCFURL):
* Source/WebKit/Shared/API/c/mac/WKWebArchiveRef.cpp:
(WKWebArchiveCreate):
(WKWebArchiveCreateWithData):
(WKWebArchiveCreateFromRange):
(WKWebArchiveCopyMainResource):
(WKWebArchiveCopySubresources):
(WKWebArchiveCopySubframeArchives):
(WKWebArchiveCopyData):
* Source/WebKit/Shared/API/c/mac/WKWebArchiveResource.cpp:
(WKWebArchiveResourceCreate):
(WKWebArchiveResourceCopyData):
(WKWebArchiveResourceCopyURL):
(WKWebArchiveResourceCopyMIMEType):
(WKWebArchiveResourceCopyTextEncoding):
* Source/WebKit/Shared/APIWebArchiveResource.h:
* Source/WebKit/Shared/BlobDataFileReferenceWithSandboxExtension.h:
* Source/WebKit/Shared/Cocoa/WKNSData.mm:
(-[WKNSData dealloc]):
* Source/WebKit/Shared/Cocoa/WKNSString.mm:
* Source/WebKit/Shared/Cocoa/WKNSURL.mm:
(-[WKNSURL _web_createTarget]):
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithInProcessRenderingBackingStore.mm:
(WebKit::RemoteLayerWithInProcessRenderingBackingStore::frontBufferMayBeVolatile const):
(WebKit::RemoteLayerWithInProcessRenderingBackingStore::createContextAndPaintContents):
(WebKit::RemoteLayerWithInProcessRenderingBackingStore::createFlusher):
(WebKit::RemoteLayerWithInProcessRenderingBackingStore::setBufferNonVolatile):
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithRemoteRenderingBackingStore.mm:
(WebKit::RemoteLayerWithRemoteRenderingBackingStore::RemoteLayerWithRemoteRenderingBackingStore):
* Source/WebKit/Shared/WebBackForwardListItem.h:
(isType):
* Source/WebKit/Shared/WebGeolocationPosition.h:
(isType):
* Source/WebKit/UIProcess/API/APIApplicationManifest.h:
* Source/WebKit/UIProcess/API/APIAttachment.h:
* Source/WebKit/UIProcess/API/APIContentRuleList.h:
* Source/WebKit/UIProcess/API/APIContentRuleListAction.h:
* Source/WebKit/UIProcess/API/APIContentRuleListStore.h:
* Source/WebKit/UIProcess/API/APIContentWorld.h:
* Source/WebKit/UIProcess/API/APIContextMenuElementInfo.h:
* Source/WebKit/UIProcess/API/APIContextMenuElementInfoMac.h:
* Source/WebKit/UIProcess/API/APIDataTask.h:
* Source/WebKit/UIProcess/API/APIFeature.h:
* Source/WebKit/UIProcess/API/APIFrameInfo.h:
* Source/WebKit/UIProcess/API/APIFrameTreeNode.h:
* Source/WebKit/UIProcess/API/APIHTTPCookieStore.h:
* Source/WebKit/UIProcess/API/APIHitTestResult.h:
* Source/WebKit/UIProcess/API/APIInspectorExtension.h:
* Source/WebKit/UIProcess/API/APIMessageListener.h:
* Source/WebKit/UIProcess/API/APINavigation.h:
* Source/WebKit/UIProcess/API/APINavigationAction.h:
* Source/WebKit/UIProcess/API/APINavigationData.h:
* Source/WebKit/UIProcess/API/APINavigationResponse.h:
* Source/WebKit/UIProcess/API/APIOpenPanelParameters.h:
* Source/WebKit/UIProcess/API/APIPageConfiguration.h:
* Source/WebKit/UIProcess/API/APIProcessPoolConfiguration.h:
* Source/WebKit/UIProcess/API/APIResourceLoadInfo.h:
* Source/WebKit/UIProcess/API/APIResourceLoadStatisticsFirstParty.h:
* Source/WebKit/UIProcess/API/APIResourceLoadStatisticsThirdParty.h:
* Source/WebKit/UIProcess/API/APISessionState.h:
* Source/WebKit/UIProcess/API/APITargetedElementInfo.h:
* Source/WebKit/UIProcess/API/APITargetedElementRequest.h:
* Source/WebKit/UIProcess/API/APITextRun.h:
* Source/WebKit/UIProcess/API/APIUserInitiatedAction.h:
* Source/WebKit/UIProcess/API/APIWebAuthenticationPanel.h:
* Source/WebKit/UIProcess/API/APIWebPushDaemonConnection.h:
* Source/WebKit/UIProcess/API/APIWebPushMessage.h:
* Source/WebKit/UIProcess/API/APIWebPushSubscriptionData.h:
* Source/WebKit/UIProcess/API/APIWebsiteDataRecord.h:
* Source/WebKit/UIProcess/API/APIWebsitePolicies.h:
* Source/WebKit/UIProcess/API/APIWindowFeatures.h:
* Source/WebKit/UIProcess/Authentication/AuthenticationChallengeProxy.h:
(isType):
* Source/WebKit/UIProcess/Authentication/AuthenticationDecisionListener.h:
(isType):
* Source/WebKit/UIProcess/Authentication/WebCredential.h:
(isType):
* Source/WebKit/UIProcess/Authentication/WebProtectionSpace.h:
(isType):
* Source/WebKit/UIProcess/Downloads/DownloadProxy.h:
(isType):
* Source/WebKit/UIProcess/GeolocationPermissionRequestProxy.h:
(isType):
* Source/WebKit/UIProcess/Inspector/WebInspectorUIProxy.h:
(isType):
* Source/WebKit/UIProcess/MediaKeySystemPermissionRequest.h:
(isType):
* Source/WebKit/UIProcess/Notifications/WebNotification.h:
(isType):
* Source/WebKit/UIProcess/QueryPermissionResultCallback.h:
(isType):
* Source/WebKit/UIProcess/SpeechRecognitionPermissionRequest.h:
(isType):
* Source/WebKit/UIProcess/UserContent/WebUserContentControllerProxy.h:
(isType):
* Source/WebKit/UIProcess/UserMediaPermissionCheckProxy.h:
(isType):
* Source/WebKit/UIProcess/UserMediaPermissionRequestProxy.h:
(isType):
* Source/WebKit/UIProcess/WebBackForwardList.h:
(isType):
* Source/WebKit/UIProcess/WebContextMenuListenerProxy.h:
(isType):
* Source/WebKit/UIProcess/WebFormSubmissionListenerProxy.h:
(isType):
* Source/WebKit/UIProcess/WebFramePolicyListenerProxy.h:
(isType):
* Source/WebKit/UIProcess/WebGeolocationManagerProxy.h:
(isType):
* Source/WebKit/UIProcess/WebPreferences.h:
(isType):
* Source/WebKit/UIProcess/WebProcessPool.h:
(isType):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h:
(isType):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.h:
(isType):
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeContext.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeContext.mm:
(WebKit::RemoteLayerTreeContext::ensureProtectedRemoteRenderingBackendProxy):

Canonical link: https://commits.webkit.org/292130@main
